source/domains/ssl/letsencrypt/letsencrypt_how.md

```diff
@@ -0,0 +1,66 @@
+# How Let's Encrypt works
+
+`Let's Encrypt` uses a 2 step process to  issue a certificate - **domain validation** (to prove you own the domain) and then the ability to issue/renew/revoke certificates thereafter.
+
+There a few distinct types of domain validation available, so you will need to first assess which method best suits your application/needs.
+
+```eval_rst
+.. warning::
+  Currently, if your sites/services  DDoSX/Webcel/WAF you will not be able to use Let's Encrypt certificates
```

Comment:
Missing words?

Reply:
missing 'use' or 'are behind'

Reply:
"If your sites/services use our DDoSX/Webcel/WAF services you will not be able to use Let's Encrypt certificates currently."